docs/pages/index.vue

```diff
@@ -64,7 +64,7 @@
         In the meantime, we welcome direct feedback and contributions! It's important that the Design System works for you, so please help shape it.
       </p>
       <ul>
-        <li>Share ideas or concerns with Devon, Jessica, and Jacob directly</li>
+        <li>Share ideas or concerns with Devon, Jessica, Khang, and Jacob directly</li>
```

Comment:
I wonder if our names should link to some way to reach out to us. Maybe our Github tags or emails?

This makes sense for internal use but community users won't know who those people are.

Reply:
```suggestion
        <li>Share ideas or concerns with Devon, Jessica, Khang, and Jacob directly, or file issues in this repo.</li>
```